api-inference-community/api_inference_community/validation.py

```diff
@@ -276,7 +277,7 @@ def ffmpeg_read(bpayload: bytes, sampling_rate: int) -> np.array:
     ffmpeg_command = [
         "ffmpeg",
         "-i",
-        "pipe:0",
+        bpayload,
```

Comment:
@Narsil this was the cause of the `Malformed soundfile` error 😅 

@osanseviero are you able to re-deploy this branch so we can use it with `superb`?

Reply:
I don't have deployment powers, but we can ask people in the infra team to do the deployment if not

Reply:
This fix is not correct unfortunately.

We really don't want to read the filename, but it really means that this doesn't run in your case:
https://github.com/huggingface/huggingface_hub/pull/242/files#diff-db8bbac95c077540d79900384cfd524d451e629275cbb5de7a31fc1cd5d6c189R319

ffmpeg_read is really correct the way it is, and it's important to keep it that way for use in API mode, where the bytes are part of the requests (they are raw bytes and need to be sent as pipe:0)

Do you mind checking why ? (Maybe extension list does not contain all needed)

Reply:
thanks for the context @Narsil! i did not appreciate that we don't want to read the filenames. 

i've reverted the change and pushed a fix in `AUDIO_EXTENSIONS` [here](https://github.com/huggingface/huggingface_hub/pull/242#discussion_r681985597).

Reply:
For info, turned out, for some reason the dataset was corrupted on drive and even though datasets was thought to be in cache, the actual audio files were not there.